src/Ionide.ProjInfo/Utils.fs

```diff
@@ -1,15 +1,22 @@
 namespace Ionide.ProjInfo
 
-module internal Paths =
+module Paths =
     /// provides the path to the `dotnet` binary running this library, duplicated from
     /// https://github.com/dotnet/sdk/blob/b91b88aec2684e3d2988df8d838d3aa3c6240a35/src/Cli/Microsoft.DotNet.Cli.Utils/Muxer.cs#L39
     let dotnetRoot =
-        System
-            .Diagnostics
-            .Process
-            .GetCurrentProcess()
-            .MainModule
-            .FileName
+        match System.Environment.GetEnvironmentVariable "DOTNET_HOST_PATH" with
```

Comment:
we add this env variable access in addition to the muxer lookup so that this member works when run during the tests.